### ISSUE TITLE
Uncaught TypeError: _this.scrollDown is not a function

### ISSUE THREAD

**Issue by alflanagan** [atom/markdown-preview#464]
[Enter steps to reproduce:]

Unknown, sorry. Had preview pane open, but was editing another window, and not the source for the preview.

**Atom**: 1.14.0-beta3 x64
**Electron**: 1.3.13
**OS**: "Fedora release
**Thrown From**: [markdown-preview](https://github.com/atom/markdown-preview) package 0.159.3

### Stack Trace

Uncaught TypeError: _this.scrollDown is not a function

```
At /usr/share/atom-beta/resources/app.asar/node_modules/markdown-preview/lib/markdown-preview-view.js:172

TypeError: _this.scrollDown is not a function
    at /app.asar/node_modules/markdown-preview/lib/markdown-preview-view.js:172:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:28.7.0 core:select-down (input.hidden-input)
  5x -0:28.1.0 core:move-down (input.hidden-input)
     -0:27 core:select-down (input.hidden-input)
     -0:25.9.0 emacs-plus:kill-region (input.hidden-input)
  4x -0:25.3.0 core:move-up (input.hidden-input)
     -0:24 core:paste (input.hidden-input)
  2x -0:23 emacs-plus:kill-line (input.hidden-input)
     -0:21.6.0 window:focus-next-pane (input.hidden-input)
     -0:21.4.0 core:move-down (div.markdown-preview.native-key-bindings)
     -0:09.1.0 core:select-up (input.hidden-input)
     -0:08.3.0 core:move-down (input.hidden-input)
  2x -0:07.8.0 core:move-up (input.hidden-input)
     -0:07.1.0 core:move-down (input.hidden-input)
  2x -0:06.6.0 core:select-up (input.hidden-input)
     -0:05.9.0 core:delete (input.hidden-input)
     -0:03.7.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.5 
atom-beautify 0.29.17 
atom-keyboard-macros 0.9.2 
atom-live-grammar-reload 0.8.0 
atom-macros 0.5.2 
atom-ternjs 0.17.0 
autoclose-html 0.23.0 
autocomplete-paths 1.0.5 
autocomplete-python 1.8.30 
clipboard-plus 0.5.1 
color-picker 2.2.4 
custom-folds 1.9.3 
dalek 0.1.0 
default-encoding 0.8.0 
disable-keybindings 0.6.0 
django-templates 0.5.0 
editorconfig 2.2.0 
emacs-plus 0.11.1 
file-icons 2.0.13 
flex-tool-bar 0.11.0 
git-diff-details 1.4.0 
git-plus 7.2.0 
grammar-mode 1.1.0 
Hydrogen 1.5.4 
ink 0.6.2 
js-hyperclick 1.10.1 
julia-client 0.5.11 
language-forth 0.2.3 
language-julia 0.10.0 
language-mips 0.3.0 
language-powershell 4.0.0 
language-utl 0.1.0 
last-cursor-position 0.9.0 
latex-completions 0.3.2 
linter 1.11.21 
linter-coffeescript 1.0.0 
linter-eslint 8.1.0 
linter-less 2.6.0 
linter-python-pyflakes 0.1.0 
linter-shellcheck 1.4.3 
linter-xmllint 1.4.0 
MagicPython 1.0.3 
minimap 4.26.8 
minimap-git-diff 4.3.1 
package-cop 0.2.10 
project-manager 3.3.3 
python-snippets 1.9.0 
python-yapf 0.13.0 
script 3.14.1 
shrink-whitespace 0.2.3 
significant-other 0.2.0 
slickedit-select 1.6.1 
sort-lines 0.14.0 
split-diff 1.1.1 
terminal-plus 0.14.5 
tool-bar 1.0.1 
uber-juno 0.1.1 
```

**Comment by Ben3eeE** [comment]
To reproduce run `core:move-down` or `core:move-up` from the `command-palette` with the `markdown-preview` pane open.

**Comment by as-cii** [comment]
Thanks for the report!

@Ben3eeE: can you reproduce on the latest version of this package? I think this issue got fixed in #463 and the fix for it has already been cherry-picked onto `1.13-releases` and `1.14-releases`.

**Comment by as-cii** [comment]
It turns out I was wrong and this wasn't fixed by #463. A fix for this has just landed on master and it will be published in the next stable and beta releases.
